adapters/Unity/UnityAdapter/GADMAdapterUnityBannerAd.m

```diff
@@ -58,30 +58,21 @@ - (void)getBannerWithSize:(GADAdSize)adSize {
     return;
   }
 
-  NSError *error = nil;
-  if (error) {
-    [strongConnector adapter:strongAdapter didFailAd:error];
-    return;
-  }
-
-  _placementID = [[[strongConnector credentials] objectForKey:kGADMAdapterUnityPlacementID] copy];
+  _placementID = [strongConnector.credentials[kGADMAdapterUnityPlacementID] copy];
 
   if (!_placementID){
     NSString *description = [NSString
-                             stringWithFormat:@"Tried to initialize %@ with nil placement.", NSStringFromClass([UADSBannerView class])];
+                             stringWithFormat:@"Tried to initialize %@ with nil placement.", @"Unity banner"];
```

Comment:
`[NSString stringWithFormat:@"Tried to initialize Unity banner with nil placement."];`

Reply:
Ignore the previous comment. Use the string directly., no need of format here.

Reply:
Removed string format and use string directly. @maninomula